src/CMakeLists.txt

```diff
@@ -31,10 +31,40 @@
 
 project(vltrace C)
 
+include(CheckCSourceRuns)
+
 add_cstyle(src)
 
 find_package(PkgConfig)
-pkg_check_modules(LIBBCC REQUIRED libbcc)
+
+if(PKG_CONFIG_FOUND)
+	pkg_check_modules(LIBBCC REQUIRED libbcc)
+else()
+	find_package(LIBBCC QUIET REQUIRED)
+endif()
+
+if(LIBBCC_FOUND)
+	set(ORIG_CMAKE_REQUIRED_LIBRARIES ${CMAKE_REQUIRED_LIBRARIES})
+	set(CMAKE_REQUIRED_LIBRARIES "${ORIG_CMAKE_REQUIRED_LIBRARIES} -lbcc")
```

Comment:
Well, if you save the original value, then probably you meant to add this after the check:
`set(CMAKE_REQUIRED_LIBRARIES "${ORIG_CMAKE_REQUIRED_LIBRARIES}")`